### ISSUE TITLE
Dockerize ELI5/long-form question answering

### ISSUE THREAD

**Issue by aolney** [memphis-iis/mofacts-ies#1069]
- [ ] Investigating using LoRA and `accelerate` to reduce latency/shrink model size
- [ ] Dockerize

**Comment by aolney** [comment]
So apparently I created a docker image of this 2 years ago and it's running on my server:

docker pull aolney/longform-qa-service:1.0

This is FYI as that version would not be optimized.

**Comment by JRustyHaner** [comment]
for this Mofacts points to localhost as well, port 5000/api for your info

**Comment by aolney** [comment]
I'm looking at an alternate method of creating elaborated feedback that, if it works, would collapse three of my current services to 1. I've asked Amanda to eyeball a few test cases to see if a full scale evaluation would be warranted.

**Comment by aolney** [comment]
See Docker  orchestration in https://github.com/memphis-iis/mofacts-ies/issues/1081

Leaving this open as the possibility of switching models is still open; if not, optimizing is  still on  the table as  well.

**Comment by aolney** [comment]
Memorializing that Amanda thought ChatGPT was better than the longformqa approach to elaborated feedback in a quick and dirty test. 

If that is an acceptable change, then there's no point in trying to optimize the existing model for size/speed - that effort could instead be spent on evaluating whether ChatGPT is reliably better (in a full human evaluation).
